STYLE.md

```diff
@@ -185,6 +185,29 @@ $ docker run --name docsbuilder \
 quay.io/coreosinc/coreos-pages-builder scripts/deploy stage
 ```
 
+### Example: Documenting Code Blocks
+
+Code blocks are similar to long command lines, but with a fenced look. To format a code block in a document, insert triple backticks ` ``` ``` `before and after the code block. Use language identifier to enable language detection and syntax highlighting in the code block. Do not add a newline after the code block, Markdown automatically does for you.
```

Comment:
Replace with: "To format a code block in a document, insert triple backticks ` ``` ``` `on a new line before and after the code block. A tag such as `yaml`, `sh`, `json`, or `ini` can be placed after the opening backticks to declare the language in the block."

and check the escaping of the literal ``` chars with the preview.